src/components/input-panel/spec-editor-header/index.css

```diff
@@ -0,0 +1,3 @@
+.spec-editor-header {
+  cursor: unset !important;
```

Comment:
Don't use important. Make sure the default is not a pointer.

Reply:
We need pane-header styles which also has higher priority cursor:pointer applied to it, so we need to mark this important otherwise these styles won't kick in